### DISCUSSION TITLE
Login Methods

### DISCUSSION THREAD

**Discussion by kaaax0815** [coder/code-server#2581] [Q&A]
What are available login methods different then password?
Is there an option for Github, Google Login etc?

**Comment by code-asher** [comment]
We only support password authentication.

For external authentication we recommend using some kind of proxy. See
the FAQ for more information: https://github.com/cdr/code-server/blob/v3.8.0/doc/FAQ.md#how-should-i-expose-code-server-to-the-internet

**Reply by code-asher** [reply]
You can also use `--link` to proxy through our cloud which will use GitHub authentication and handle all the proxying and TLS for you.
